tensorflow_serving/servables/tensorflow/util.cc

```diff
@@ -47,6 +47,48 @@ auto* example_count_total = monitoring::Counter<1>::New(
     "/tensorflow/serving/request_example_count_total",
     "The total number of tensorflow.Examples.", "model");
 
+// Metrics by model
+auto* model_request_count_total = monitoring::Counter<1>::New(
+    "/tensorflow/serving/model_request_count",
+    "The total number of requests.", "model");
+
+auto* model_request_fail_count_total = monitoring::Counter<1>::New(
```

Comment:
remove this if you are tracking per-model+error-code counts in previous counter.